tools/driver/CMakeLists.txt

```diff
@@ -13,6 +13,21 @@ target_link_libraries(swift-frontend
                         swiftSymbolGraphGen
                         LLVMBitstreamReader)
 
+# Generate and install supported feature JSON file to the toolchain.
+if(NOT CMAKE_SYSTEM_NAME STREQUAL "Windows")
+  set(DUMMY_SWIFT_INPUT "${SWIFT_RUNTIME_OUTPUT_INTDIR}/dummyInput.swift")
+  set(INT_FEATURE_FILE "${SWIFT_RUNTIME_OUTPUT_INTDIR}/SupportedFeatures.json")
+  add_custom_command(TARGET swift-frontend POST_BUILD
+                     COMMAND "touch" "${DUMMY_SWIFT_INPUT}"
```

Comment:
Can you use `${CMAKE_COMMAND} -E touch` instead please?

Reply:
Fixed.